src/Native/Platform.js

```diff
@@ -528,6 +530,11 @@ function setupIncomingPort(name, callback)
 	function onEffects(router, subList, state)
 	{
 		subs = subList;
+		if (!onEffectsCalled) {
+			onEffectsCalled = true;
+			enqueuedBeforeSetup.forEach(internalSend);
+			enqueuedBeforeSetup = null;
```

Comment:
should this be turned into an empty array again?  Feels weird changing its type.